src/PolykeyAgent.ts

```diff
@@ -401,11 +405,15 @@ class PolykeyAgent {
       // If a recovery code is provided then we reset any sessions in case the
       //  password changed.
       if (keyRingConfig.recoveryCode != null) await sessionManager.resetKey();
-      grpcServerClient =
-        grpcServerClient ??
-        new GRPCServer({
-          logger: logger.getChild(GRPCServer.name + 'Client'),
-        });
+      // FIXME: propagate default values
+      webSocketServer = new WebSocketServer(
+        logger.getChild('WebSocketServer'),
+        fs,
+        1_000_000_000, // About 1 GB
+        undefined,
+        1_000,
+        10_000,
+      );
```

Comment:
This should be DIed like `webSocketServer = webSocketServer ?? new WebSocketServer(...)`.

Also what do you mean by propagating the default values?

Reply:
There are config values that are just hard-coded here. They need to be added to the `PolykeyAgent` config and defaults.

Reply:
This has been changed.